pkgs/applications/science/biology/bftools/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, lib, pkgs, fetchurl, unzip, openjdk }:
+
+stdenv.mkDerivation rec {
+  name = "bftools-${version}";
+  version = "5.9.2";
+
+  src = fetchurl {
+    url = "http://downloads.openmicroscopy.org/bio-formats/${version}/artifacts/bftools.zip";
+    sha256 = "02x8p07z7js41kgiblkr9k8v9v4ka8rm7pg3f1vy9s5p6hmpfvnx";
+  };
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./* $out/
+    for script in $(find . -maxdepth 1 -perm -111 -type f); do
+      ln -s $out/$script $out/bin/$script
+    done
+  '';
+
+  postFixup = ''
+        for script in "$out"/bin/*; do
+          wrapProgram "$script" --prefix PATH : "${lib.makeBinPath [ openjdk ]}"
+        done
+      '';
+
+  buildInputs = [ unzip pkgs.makeWrapper ];
```

Comment:
pkgs.makeWrapper should go to `nativeBuildInputs`

Reply:
This is funny. I don't understand too much of this either.

In essence, things that you put into `nativeBuildInputs` will be some native utilities that are supposed to run during the build (so they are "native" to the architecture where build executes) and nativeBuildInputs is for dependencies (in case of cross compilation they will be the ones that were also compiled for the other architecture).

Reply:
quick clarification, do you mean "buildInputs is for dependencies"?

Reply:
@tbenst Yes, corrected, thanks.